fract4d/c/model/color.h

```diff
@@ -1,12 +1,20 @@
-
 #ifndef COLOR_H_
 #define COLOR_H_
 
 struct s_rgba
 {
     unsigned char r,g,b,a;
+
+    inline explicit operator int() const {
+        return (r << 16) | (g << 8) | b;
+    }
 };
 
+inline bool operator== (const s_rgba& a, const s_rgba& b) {
+    // according to the original implementation "a" is not considered in the comparison
```

Comment:
I can't honestly remember why that would be 🤔

Reply:
I think it's because the alpha it's not saved in the image buffer:
```C++
rgba_t image::get(int x, int y) const
{
    char *start = buffer + x * 3 + y * m_Xres * 3;
    //assert(start  + 2 - buffer <= bytes());
    rgba_t pixel;
    pixel.r = start[RED];
    pixel.g = start[GREEN];
    pixel.b = start[BLUE];
    pixel.a = 0;
    return pixel;
}
```